modules/videopress/class.videopress-xmlrpc.php

```diff
@@ -155,18 +155,25 @@ public function update_poster_image( $request ) {
 			return false;
 		}
 
+		// Switch to photon if needed.
+		$poster = str_replace(
+			'https://videos.files.wordpress.com',
+			'https://i0.wp.com/videos.files.wordpress.com',
```

Comment:
Should we use `apply_filters( 'jetpack_photon_url`, $poster )` here, so all Photon filters / domain sharding / etc can be applied if needed?